src/wallet/wallet.cpp

```diff
@@ -167,6 +167,10 @@ bool CWallet::AddSaplingZKey(
     }
 
     // TODO: Persist to disk
```

Comment:
Is this comment stale?

Reply:
Yes, it was used as a place holder for code implementation. I will clean this up for final pass.